service/src/main/java/aero/minova/cas/service/QueueService.java

```diff
@@ -227,88 +252,117 @@ private void saveMessage(String message, String procedureName, String topic) {
 	}
 
 	/**
-	 * Löscht eine Nachricht aus der Datenbank, damit nicht mehr versucht wird sie zu versenden.
+	 * Löscht eine Nachricht aus der Datenbank, damit nicht mehr versucht wird sie zu versenden. NEUERUNG: Wir löschen Nachrichten nicht mehr, sondern setzen
```

Comment:
Was ist der Unterschied zu LastAction und Failed?
Nachrichten, die älter als 30 Tage sind, sollten zudem gelöscht werden.

Reply:
Ok, dann habe ich das falsch verstanden, dann werde ich Nachrichten wieder löschen lassen.

An sich der Unterschied zwischen Failed und LastAction ist nur, dass Failed ein Boolean ist und LastAction ein int.
Kann sein, dass ich das in unserer Konversation auch falsch verstanden habe und deshalb Failed eingeführt habe. Wenn du möchtest, kann ich den Parameter raus nehmen und die Prüfungen auf LastAction abändern.